dynoid/dynoid.go

```diff
@@ -5,43 +5,234 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"log/slog"
+	"os"
+	"path"
+	"strconv"
 	"strings"
 	"sync"
 
 	"github.com/coreos/go-oidc/v3/oidc"
 )
 
-type IssuerCallback func(string) error
+const (
+	AudienceHeroku = "heroku"
+)
```

Comment:
Each service is supposed to use a unique audience so defining this as a constant might send the wrong signal.

Reply:
What if I just don't export the constant?